microsoft-edge/dev-videos/index.md

```diff
@@ -29,6 +29,22 @@ To add a new video:
 -->
 
 
+<!-- ====================================================================== -->
+## Web Platform - Create a scroll-linked animation without JavaScript
+
+_Published on 12 May 2022._
+
+[![Thumbnail image for the scroll-linked animation API video](./images/scroll-linked-animations.png)](https://www.youtube.com/watch?v=Q0nhiHVVnvI)
+
+Learn about the upcoming CSS scroll-linked animations feature and how it can be used to create a reading progress indicator on a web page without JavaScript.
+
+This feature is experimentally supported in Microsoft Edge and you can play with it by enabling the "Experimental Web Platform features" flag in `edge://flags`.
```

Comment:
Acrolinx (try it & see) would probably suggest breaking up into two sentences; two distinct points are being made.   This is an opportunity to lower the reading grade level.  eg:

This feature is experimentally supported in Microsoft Edge.  To try this feature, enable the "Experimental Web Platform features" flag in `edge://flags`.
```suggestion
This feature is an experimental feature in Microsoft Edge.  To try this feature, go to `edge://flags` and then enable the **Experimental Web Platform features** setting.
```